VideoWeb/VideoWeb/ClientApp/src/app/shared/chat/chat-base.component.spec.ts

```diff
@@ -34,9 +34,15 @@ class ChatBaseTest extends ChatBaseComponent {
     sendMessage(messageBody: string): void {
         this.messagesSent.push(messageBody);
     }
+
     get participantUsername(): string {
```

Comment:
I wonder if this is still needed.

Reply:
Yes, we need it in participant-chat component